### ISSUE TITLE
Static weaving plugin-dependencies for Java 11

### ISSUE THREAD

**Issue by jonnybecker** [ethlo/eclipselink-maven-plugin#27]
Static weaving during build only works with Java 11, when I add an extra dependeny to the `eclipselink-maven-plugin` plugin. 
 
```
<pluginManagement>
  <plugins>
	<plugin>
	  <groupId>com.ethlo.persistence.tools</groupId>
	  <artifactId>eclipselink-maven-plugin</artifactId>
	  <version>2.7.4.1</version>
	  <dependencies>
		<dependency>
		  <groupId>org.glassfish.jaxb</groupId>
		  <artifactId>jaxb-runtime</artifactId>
		  <version>2.3.2</version>
		</dependency>
	  </dependencies>
	</plugin>
        ...
```

Since this dependency (which I'm not sure is the correct one, since its a complete bundle of libs) is only needed during build it should be used by the plugin itself (for Java 11).

**Comment by ethlo** [comment]
Thanks for the report.

**Comment by ethlo** [comment]
Fixed in 2.7.5
